std_msgs/Header header
exploration/ScanPoint[] points
